__tests__/api/utils.test.js

```diff
@@ -32,44 +49,77 @@ describe('Utils', () => {
     });
   });
 
+  describe('createReverseRelationships()', () => {
+    const state = {
+      resources: {
+        people: {
+          9: {
+            relationships: {
+              comments: {
+                data: [
+                  { id: '12', type: 'comments' }
+                ]
+              }
+            }
+          }
+        },
+        comments: {
+          5: {},
+          12: {}
+        },
+      }
+    };
+
+    test('updates hasMany relationships', () => {
+      const newState = imm(state);
+      utils.createReverseRelationships(state, newState, comments[0]);
```

Comment:
Thinking it may be helpful to have the comment in this file or at least in a more obvious fixture. If in this file, we don't have to leave to see what data structures we're working with